### DISCUSSION TITLE
Was parallel support ever added for the ESP32-S2?

### DISCUSSION THREAD

**Discussion by JimDrewGH** [Bodmer/TFT_eSPI#1940] [Q&A - General]
The title says it all.  I have a case where I am stuck using an ESP32-S2 because I need a couple of extra I/O pins, but I have a parallel (only) display.  I can't find any information about parallel display support for the ESP32-S2 other than in March about looking into it.

Thanks!

**Comment by Bodmer** [comment]
I have not tested the S2 with a parallel display but it should work.

C3 and S3 have been tested with 8 bit parallel interface. S2, C3 and S3 with SPI interface.

**Reply by JimDrewGH** [reply]
What about the S2?

**Reply by Bodmer** [reply]
Meant S2! Have corrected reply above.

**Reply by JimDrewGH** [reply]
Thanks.  I have displays that just arrived so I will test and report back.

**Reply by JimDrewGH** [reply]
I just posted a bug report as I am not able to successfully compile the Boing-ball demo with the ESP32-S2.

**Comment by Bodmer** [comment]
DMA is only supported on original ESP32 at the moment.

**Reply by JimDrewGH** [reply]
Ok, thanks for confirming.  Let me know if you want something tested.  I have developer board here that I am forced to use LovyanGFX with because it has S2 DMA support.  I am not too thrilled about it because I am not familiar with it and I had to spend the day translating Kanji just to get something to work.  :)
